packages/compass/src/main/application.ts

```diff
@@ -98,6 +98,17 @@ class CompassApplication {
       );
     }
 
+    const IS_CI =
+      process.env.IS_CI ||
+      process.env.CI ||
```

Comment:
Sorry for being hair splitty, but why do we rely on such a generic environment variable, especially no linux where i'm even more likely to start compass from a terminal. For example, as a developer i may have this variable set on my machine to test something out, and regardless, we should not rely on non production environment variable in a way that may affect production this much. 

Is "EVERGREEN_BUILD_VARIANT" here not enough? possibly better, did you consider a more intentional variable to be set explicitly on CI, for example: `MONGODB_COMPASS_USE_PLAIN_SAFE_STORAGE=true`?

Reply:
Actually `EVERGREEN_BUILD_VARIANT` should suffice this check, however its being used throughout along with `CI` for ci check. I wanted to keep the behaviour. 
For `IS_CI`, I think we can remove it as I can not find where/how we set it.

And as for adding a new variable, I did not consider that because we already have CI checks in place and wanted to have similar check here as well.

Reply:
do you mean maybe: `EVERGREEN_BUILD_VARIANT && CI` then rather than `||`?

Reply:
I was thinking of having similar behaviour as we have right now and that's using `||`. But the fact that users can have it set in local env will have unexpected behaviour, it makes sense either to `&&` it or use an explicit variable name. I am fine any from these two options. Let me know what you think?

Reply:
We usually use `MONGODB_COMPASS_TEST_` for this kind of stuff, we should probably stick to the pattern? Generally speaking yeah, a bit weird to see `IS_CI` env checks right in the Compass app source, I'd probably have a preference for avoiding this

Reply:
added [fb3dedf](https://github.com/mongodb-js/compass/pull/5214/commits/fb3dedf2aaa877b6ae810cc5192cfab80c962e16)